pages/data-providers/[data-provider-id]/settings.css

```diff
@@ -4,4 +4,8 @@
 
 .section {
   margin-bottom: calc(1.5 * var(--component-padding-y, 1rem));
+
+  & input {
+    max-width: 50rem;
```

Comment:
You use this value twice. Is not it better to apply this to form as a class and then apply it the iframe.

Reply:
it's going to be improved in https://github.com/oacore/dashboard/pull/263. I will leave it as is here for the time being.